eksctl-scripts/update-nodegroups.sh

```diff
@@ -23,17 +23,29 @@ CLUSTER_NAME=$(cat $CLUSTER_FILE | yq r - metadata.name)
 
 eksctl get nodegroup --cluster $CLUSTER_NAME
 
-OLD_COMMIT_ID=$(cut -d'v' -f 2 <<< $(eksctl get nodegroup --cluster $CLUSTER_NAME | grep $CLUSTER_NAME | awk '{ print $2; }' | head -n 1))
-sed -i "s/-v[0-9a-f]\+$/-v$OLD_COMMIT_ID/g" $CLUSTER_FILE
+# Managed Nodegroups
+if [ "$(yq r $CLUSTER_FILE 'managedNodeGroups' | grep 'name' || echo "nope")" == "nope" ]; then
+  echo "No managed nodegroups found"
+else
+  # TODO: Config changes upgrade is only partially supported, as far as I understand it
+  # unchanged fields for nodegroup managed-group-a: the following fields remain unchanged; they are not supported by `eksctl update nodegroup`: AvailabilityZones, ScalingConfig, VolumeSize, SSH, PrivateNetworking, Tags, IAM, VolumeType, VolumeEncrypted, InstanceTypes, Spot, Taints
+  eksctl update nodegroup -f $CLUSTER_FILE
+  yq r $CLUSTER_FILE 'managedNodeGroups' | grep name | awk '{ print $3; }' | xargs -i eksctl upgrade nodegroup --cluster=$CLUSTER_NAME --name={}
```

Comment:
Wieso update nodegroup & upgrade nodegroup? Könnte man update nodegroup auch pro group machen?

Reply:
Ich glaube "eksctl update nodegroup -f $CLUSTER_FILE" brauchen wir an der Stelle eh nicht, da wir ja in der Pipeline schon ein "Upgrade" wollen. Derzeit supported es nur die "upgrade configuration": 
Right now, it will only update upgrade configuration.